src/Compilers/Core/Portable/Compilation/ParseOptions.cs

```diff
@@ -37,7 +37,10 @@ public ParseOptions WithKind(SourceCodeKind kind)
             return CommonWithKind(kind);
         }
 
-        public abstract ParseOptions CommonWithKind(SourceCodeKind kind);
+        public virtual ParseOptions CommonWithKind(SourceCodeKind kind)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
The use of `abstract` is not a breaking change here because this type is not generally extendable.  The type has only `internal` constructors and all of the types we produce that derive from it are `sealed`.